oddbird/Metecho
# [toast-messages] more precise language for org toast messages

## Cute animal pic
![image](https://user-images.githubusercontent.com/41588129/120654971-7a34b800-c482-11eb-90bb-b9ce4e416153.png)

_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._



## Link to Trello card (if applicable)
https://trello.com/c/sW6isTAD/285-toast-messages-about-scratch-orgs
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._



## Description
_The commit messages say what you did; this should explain why and/or how._
- [x ] Description is in Trello card



## Steps to test/reproduce
Messages about orgs should be specific and say Scratch/Test/Dev depending on the type of org a message is referring to. 
- Create, delete or make changes in an org.
- The toast message should say 'Test', 'Dev' or 'Scratch' org for each respective type.
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._



## Show me


![image](https://user-images.githubusercontent.com/41588129/120655095-9a647700-c482-11eb-9c9c-63ee6bf61b64.png)

![image](https://user-images.githubusercontent.com/41588129/121040557-48895d00-c7b2-11eb-9026-c1ccab8499d1.png)

![image](https://user-images.githubusercontent.com/41588129/121040709-6951b280-c7b2-11eb-9735-eac6d7710a28.png)

![image](https://user-images.githubusercontent.com/41588129/121040828-82f2fa00-c7b2-11eb-957d-d962b071b1d4.png)



_Provide screenshots/animated gifs/videos if necessary._
